lib/logflare_web/schemas.ex

```diff
@@ -0,0 +1,60 @@
+defmodule LogflareWeb.Schemas do
```

Comment:
Let's name this to OpenApiSchemas, using Schemas by itself is too ambiguous and can be mistaken for table schemas.